engine: changes to the sidebar + landing page (#972)
redpanda-data/documentation: docs/deploy/deployment-option/self-hosted/manual/high-availability.mdx

While Apache Kafka® uses in-sync replicas, Redpanda uses a quorum-based majority with the Raft replication protocol. Kafka performance is negatively impacted when any replica is out-of-sync, but a Redpanda cluster tolerates replica failures without any performance degradation.  

> Monitor the health of your cluster with the [`rpk cluster health`](../../../../../reference/rpk/rpk-cluster/rpk-cluster-health) command. This tells you how many nodes are down (if any), as well as if you have any leaderless partitions. 

Reviewer: ```suggestion
Monitor the health of your cluster with the [`rpk cluster health`](../../../../../reference/rpk/rpk-cluster/rpk-cluster-health) command. This tells you how many nodes are down (if any), and if you have any leaderless partitions. 
```
